ArcadiaParties.Data/Data/RoleSeed.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "Name": "User"
+  },
+  {
+    "Name": "Administrator"
+  },
+  {
+    "Name": "Administrator"
```

Comment:
Why do you need two roles with the same name?

Reply:
I've deleted it